services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/configuration/Configuration.kt

```diff
@@ -55,17 +55,19 @@ class Configuration {
     fun cableReportService(webClientBuilder: WebClient.Builder): CableReportService {
         val webClient =
             if (alluTrustInsecure) createInsecureTrustingWebClient(webClientBuilder)
-            else webClientBuilder.build()
+            else webClientBuilder
```

Comment:
Should val webClient be renamed? It seems to be a builder at this point still.

Reply:
webClient is a WebClient again.